routes/confirmation/checkAnswers.js

```diff
@@ -87,6 +87,18 @@ const answerInfo = [
         urlPath: '/trillium/energy/cost/amount',
         displayIf: 'deductions.trilliumEnergyCostClaim',
       },
+      {
+        text: 'Did you use public transit in 2018?',
+        infoPath: ['deductions.seniorTransitClaim'],
+        urlPath: '/deductions/senior-public-transit',
+        displayIf: 'deductions.seniorTransitClaim',
+      },
```

Comment:
I think we want this displayed regardless (so we want to show them saying No). So no need for the displayIf on the yes/no portion

Reply:
I added some other stuff earlier today that I forgot to push up until now that probably needs to be looked over as well, and I'll confirm with Brian on this whether we want to always display it or not, I was just taking a best guess so you're probably right